queries.sql

```diff
@@ -212,4 +212,21 @@ JOIN animals a ON vs.animal_id = a.id
 JOIN species s ON s.id = a.species_id
 WHERE vt.name = 'Maisy Smith'
 GROUP BY s.name, vt.name
-ORDER BY amount DESC LIMIT 1;
\ No newline at end of file
+ORDER BY amount DESC LIMIT 1;
+
+
+
+-- ---------------- --
+-- FIFTH MILESTONE --
+-- ---------------- --
+
+SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
+
+SELECT * FROM visits WHERE vet_id = 2;
+EXPLAIN ANALYZE SELECT * FROM visits WHERE vet_id = 2;
+
```

Comment:
- [ ] Great job working on the queries so far but I think this query is still taking too much time while searching for it since this should be after `EXPLAIN ANALYZE` Lower than (1 sec = 1000ms as mentioned in the requirements. so I think you could do it many times till you get the ideal performance to get the query lower than 1 sec in the execution time 🤩🔥💪🏻

- I think it still needs more improvement ♻🔥💪🏻
![image](https://github.com/M-Anwar-Hussaini/Vet-Clinic/assets/58553711/066be83a-4eea-4da2-b4a7-00afe4f5c22b)

Reply:
Hi @Reem-lab. 
Thanks a lot for revising our project. The second picture that you have taken is correct, the query is executed more than 1 second. But the first picture you have mentioned is from the time before creating the index on the relevant column of the table.

Reply:
Hello @demesameneshoa I think both of the pictures are more than 1 sec since one of them should be more than 1 sec before Explain analyze and the second should be lower than 1 sec and this is after  Explain analyze